src/guides/v2.3/extension-dev-guide/events-and-observers.md

```diff
@@ -127,6 +127,9 @@ The `observer` [xml](https://glossary.magento.com/xml) element has the following
 *  `disabled` - Determines whether this observer is active or not. Default value is false.
 *  `shared` - Determines the [lifestyle]({{ page.baseurl }}/extension-dev-guide/build/di-xml-file.html#object-lifestyle-configuration) of the class. Default is `true`.
 
+{: .bs-callout .bs-callout-warning}
+The observer `name` must be unique. Otherwise, an override happen.
```

Comment:
Rephrase to be clearer. Maybe "The observer `name` must be unique, or an override may occur." or similar.